.pylintrc

```diff
@@ -14,3 +16,4 @@ good-names=i,
 
 [MESSAGES CONTROL]
 disable=missing-docstring,assigning-non-slot,unspecified-encoding,no-name-in-module
+
```

Comment:
Maybe try adding

```
[TYPECHECK]

ignored-modules = cv2
```

To fix the false positives

Reply:
worked thank god